stdlib/3/unittest/util.pyi

```diff
@@ -0,0 +1,29 @@
+from typing import Any, List, Sequence, Tuple, TypeVar
+
+_T = TypeVar('_T')
+_Mismatch = Tuple[_T, _T, int]
```

Comment:
Should use [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple).

Reply:
NamedTuples can't be generic though.